test-engineering/contract-tests/volumes/client/scenarios.yml

```diff
@@ -231,297 +231,297 @@ scenarios:
-            # We use a random IP address from the range specified by the CIDR network notation "216.160.83.56/29"
-            # from https://github.com/maxmind/MaxMind-DB/blob/main/source-data/GeoLite2-City-Test.json
-            # The following value will result in country-code: US and region-code: WA
-            - name: X-Forwarded-For
-              value: '216.160.83.62'
-        response:
-          status_code: 200
-          content:
-            tiles:
-              - id: 22345
-                name: 'Example COM'
-                click_url: 'https://example.com/us_wa_desktop_windows?version=16.0.0&key=22.1&ci=6.2&ctag=1612376952400200000'
-                image_url: 'https://example.com/us_wa_desktop_windows01.jpg'
-                image_size: null
-                impression_url: 'https://example.com/us_wa_desktop_windows?id=0001'
-                url: 'https://www.example.com/us_wa_desktop_windows'
-              - id: 66789
-                name: 'Example ORG'
-                click_url: 'https://example.org/us_wa_desktop_windows?version=16.0.0&key=7.2&ci=8.9&ctag=E1DE38C8972D0281F5556659A'
-                image_url: 'https://example.org/us_wa_desktop_windows02.jpg'
-                image_size: null
-                impression_url: 'https://example.org/us_wa_desktop_windows?id=0002'
-                url: 'https://www.example.org/us_wa_desktop_windows'
-      - request:
-          service: partner
-          method: GET
-          path: '/records/'
-          headers:
-            - name: 'accept'
-              value: '*/*'
-        response:
-          status_code: 200
-          content:
-            records:
-              - count: 2
-                record:
-                  method: GET
-                  headers:
-                    - name: accept
-                      value: '*/*'
-                    - name: user-agent
-                      value: 'contile/1.8.0'
-                    - name: host
-                      value: 'partner:5000'
-                  path: '/tilesp/desktop'
-                  query_parameters:
-                    - name: partner
-                      value: 'partner_id_test'
-                    - name: sub1
-                      value: 'sub1_test'
-                    - name: sub2
-                      value: 'newtab'
-                    - name: country-code
-                      value: 'US'
-                    - name: region-code
-                      value: 'WA'
-                    - name: dma-code
-                      value: '819'
-                    - name: form-factor
-                      value: 'desktop'
-                    - name: os-family
-                      value: 'windows'
-                    - name: v
-                      value: '1.0'
-                    - name: out
-                      value: 'json'
-                    - name: results
-                      value: '5'
+#  - name: legacy_image_filter
```

Comment:
We should drop this or at least add a comment block indicating that we no longer need to do image filter tests, since they're now part of the default configuration. 

Make sure to also flag context-services-test-eng as a reviewer since we're modifying the tests.

Reply:
eek, lemme go through those tests and uncomment accordinly first, I initially commented out to help me test locally.